src/public/assets/Dropdown.js

```diff
@@ -0,0 +1,147 @@
+const DATABASE = require('../../database.json');
+
+let nextID = 0;
```

Comment:
rather than nextID = 0, Dropdown.id = 0 creates a static member for it, then its more directly tied to the class. I believe they are adding the static keyword for es6 :)

Reply:
JS doesn't let you do static members just static functions :)